nixos/tests/boot.nix

```diff
@@ -114,8 +114,28 @@ in {
       cdrom = "${iso}/iso/${iso.isoName}";
     };
 
-    biosUsb = makeBootTest "bios-usb" {
+    biosUsb = let
+      seabiosNoTSC = pkgs.seabios.overrideAttrs (_: {
+        configurePhase = ''
+          cat > .config << EOF
```

Comment:
I think it would be neat to add 'tsc' and 'csm' parameters to the `seabios` derivation, but this works as well :)

Reply:
Good idea